### ISSUE TITLE
Feature request: Possibility to use RM1 buttons for Up/Down scrolling

### ISSUE THREAD

**Issue by runil** [mb1986/rm-hacks#191]
I frequently encounter unresponsiveness with scrolling touch gestures on the remarkable. Particularly when holding the pen too closely, triggering what appears to be accidental touch protection.

To enhance the tablet's usability, I suggest considering the use of RM1 left/right buttons as an extra option for up/down scrolling. In horizotal mode this would be a natural up/down press to scroll, hopefully more predictable than the touch gestures.

**Comment by ingatellent** [comment]
To me the perfect implementation would be, such that the  "next page button" scrolls a screen down, until it reaches the bottom. Then it turns to the next page. (And the other way with the other button.)

I.e. I also desire to ability to use the buttons to scrolling, but I would nog sacrifice the ability to use them as page turners.

**Comment by runil** [comment]
> To me the perfect implementation would be, such that the "next page button" scrolls a screen down, until it reaches the bottom. Then it turns to the next page. (And the other way with the other button.)
> 
> I.e. I also desire to ability to use the buttons for scrolling, but I would not sacrifice the ability to use them as page turners.
I agree that this would be great for reading zoomed in PDF's etc. However it might cause issues when writing notebooks. I often intend to scroll all the way down just to get more writing space, whereby this could cause an unintentional page change if scrolling down to the bottom and accidentally clicking one too many times. 

An alternative approach to preserve the original page-turning functionality could involve holding the button for a page change and a normal click to scroll. I suppose options for this could iplemented in a new menu, allowing users to remap the RM1 buttons to different actions similar to how the Stylus menu work.
